media/js/base/site.js

```diff
@@ -64,10 +64,7 @@
             if (pf.indexOf("Mac") !== -1) {
                 return 'oldmac';
             }
-            if (pf === '' &&
-                    /Firefox/.test(ua) &&
-                    /Mobile|Tablet/.test(ua) &&
-                    !(/Android/.test(ua))) {
+            if (pf === '' && /Firefox/.test(ua)) {
```

Comment:
`/^Mozilla\/[0-9.]+ \(.*\) Gecko\/[0-9.]+ Firefox\/[0-9.]+$/.test(navigator.userAgent)`

Gave this a quick whirl and it seems to return `true` for me for Firefox desktop and Android UA strings?